core-v1/src/main/scala/dsl/BuildExprDsl.scala

```diff
@@ -116,6 +152,34 @@ trait BuildExprDsl extends DebugExprDsl {
 
     def >=<(expr: I ~:> W[Window[V]]): I >=< V = Expr.WithinWindow(valueExpr, expr)
   }
+
+  implicit def compareIsEqual[I, V : OP](
+    valueExpr: I ~:> W[V],
+  )(implicit
+    compareV: EqualComparable[W, V, OP],
+    opV: OP[W[V]],
+    opB: OP[W[Boolean]],
+  ): EqualComparisonExprBuilder[I, V] =
+    new EqualComparisonExprBuilder(valueExpr)
+
+  class EqualComparisonExprBuilder[I, V : OP](
+    protected val leftExpr: I ~:> W[V],
+  )(implicit
+    eqV: EqualComparable[W, V, OP],
+    opV: OP[W[V]],
+    opB: OP[W[Boolean]],
+  ) {
+
+    def ===(literal: V): Expr.IsEqual[I, V, W, OP] = Expr.IsEqual(leftExpr, Expr.Const(WrapConst[W].wrapConst(literal)))
```

Comment:
Remove these literal versions to better match the `>`, `<=`, etc comparison operators